### PR TITLE
KAA-1356 Cross-compilation guide for C++ SDK (Intel Edison) on Linux platform

### PR DESCRIPTION
**Pull request by mlysenko**
Updated the previous version of the cross-compilation guide.

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 1]
```diff
@@ -115,21 +115,7 @@ The 3rd party conponents listed above have to be cross-compiled before building
 
     Copy `${SDKTARGETSYSROOT}/usr/local/lib/libboost_*` object files to Edison's `/usr/local/lib` directory (e.g. using SSH, SCP, etc.).
 
-2. Install Avro for host and target. Also Avro depends on some Boost components. So they need to be installed too.
```
Comment (vadimol) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md:118]: Why did you remove this?

**Review by vadimol — CHANGES_REQUESTED** [review]

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 2]
```diff
@@ -115,21 +115,7 @@ The 3rd party conponents listed above have to be cross-compiled before building
 
     Copy `${SDKTARGETSYSROOT}/usr/local/lib/libboost_*` object files to Edison's `/usr/local/lib` directory (e.g. using SSH, SCP, etc.).
```
Comment (vadimol) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md:116]: Please, define the SDKTARGETSYSROOT variable somewhere

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md (hunk 1 above)
Comment (mlysenko) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md:118]: I added a requirement to install Boost and Avro to the first chapter of the cross-compilation guide as a prerequisite:

> ... perform steps 1 and 2 of the [Linux guide] ... to install Boost and Avro components for your host machine.

I think it makes the guide easier to understand.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md (hunk 2 above)
Comment (mlysenko) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md:116]: `SDKTARGETSYSROOT` is defined in `environment-setup-core2-32-poky-linux` file which is generated by installer.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md (hunk 1 above)
Comment (vadimol) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md:118]: Got it, thanks

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md (hunk 2 above)
Comment (vadimol) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md:116]: Got it, thanks

**Review by vadimol — APPROVED** [review]

**Comment by vadimol** [comment]
@kaa-leeroy retest this please

**Review by vadimol — APPROVED** [review]

**Pull request merged**